docs/source/getting-started.rst

```diff
@@ -1,65 +1,30 @@
-Getting started with OAuthenticator
-===================================
-
-TODO:
-
-
-Steps
+Get started
+===========
 
+The general steps to take when using OAuthenticator:
 
 1. pick your provider
 2. register with your provider
 3. configure JupyterHub (if supported, pick oauthenticator class, otherwise see :doc:`writing-an-oauthenticator`).
 4. common configuration (client_id, client_secret, callback_url, whitelist, etc.)
 5. specific configuration for your provider
```

Comment:
- I think it would make sense to say `identity provider` throughout instead of just provider.
- I think point 3 is a bit too vague, it could be choose authenticator class or something like that instead, and avoid suggesting the creation of a custom authenticator before falling back to the generic authenticator first, and only if that isn't enough, consider creating your own. I think, not suggesting to create an own at all is best here but instead suggesting to choose a specific or the generic authenticator class.
- "common configuration" could be reworded as "configure the authenticator class" or similar to assume less from the reader.